--- a/Makefile	2021-04-11 15:08:49.516396508 -0600
+++ b/Makefile	2021-04-11 15:10:35.001403141 -0600
@@ -143,7 +143,9 @@
 	@echo removing executables from $(DESTDIR)$(PREFIX)/bin
 	@cd $(DESTDIR)$(PREFIX)/bin && rm -f $(BIN)
 	@echo removing manual pages from $(DESTDIR)$(MANPREFIX)/man1
-	@cd $(DESTDIR)$(MANPREFIX)/man1 && rm -f $(MAN)
+	@cd $(DESTDIR)$(MANPREFIX)/man1 && rm -f $(MAN1)
+	@echo removing manual pages from $(DESTDIR)$(MANPREFIX)/man8
+	@cd $(DESTDIR)$(MANPREFIX)/man8 && rm -f $(MAN8)
 
 dist: clean
 	@echo creating dist tarball
