--- a/conf/Unix.mk	2021-03-03 21:58:24.290267411 -0700
+++ b/conf/Unix.mk	2021-03-03 21:58:51.650265308 -0700
@@ -37,7 +37,7 @@
 MKDIR = mkdir -p
 
 # C compiler and flags.
-CC = cc
+CC = tcc
 CFLAGS = -W -Wall -Os -fPIC
 CCOUT = -c -o 
 
@@ -47,12 +47,12 @@
 AROUT =
 
 # DLL building tool.
-LDDLL = cc
+LDDLL = tcc
 LDDLLFLAGS = -shared
 LDDLLOUT = -o 
 
 # Static linker.
-LD = cc
+LD = tcc
 LDFLAGS = 
 LDOUT = -o 
 
